src/page-response-impact/ImpactMetricCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
```

Comment:
question: could this be something that should go into `/design-system` for potential use in other areas?

Reply:
I'm not sure, I think if there are mocks later on that use these components elsewhere it would be simple enough to extract it from here, but since I didn't see them used anywhere else and the styling seemed to be specific to this mock I kept it in this directory for now.